src/unit/test_zmalloc.c

```diff
@@ -0,0 +1,56 @@
+#include "../zmalloc.c"
```

Comment:
```suggestion
#include "../zmalloc.h"
```

In this case it looks we're just calling public interfaces, so I think we can just use zmalloc.h